src/components/card/card.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import { Link } from "gatsby";
+
+import Category from "../category/category.tsx";
+
+import * as cardStyles from "./card.module.scss";
+
+const Card = ({ data }) => (
+  <article key={data.id}>
+    <div className={cardStyles.listNode}>
```

Comment:
Maybe you can remove the div component:
```suggestion
  <article key={data.id} className={cardStyles.listNode}>
```

Reply:
This change is in the following PR, as it has reference to two bugs that I have fixed, so I will mark the comment as resolved. Is this correct for you?